tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -488,6 +488,30 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
       }
     }
 
+    it should s"invoke action which receives an empty entity (auth? ${creds.isDefined})" in {
```

Comment:
This test will have to be updated once https://github.com/apache/incubator-openwhisk/pull/2741/files is merged. Will have to add ```headers = List(`Content-Type`(ContentTypes.`application/json`))``` to `metaPayload`.